public/assets/js/scripts.js

```diff
@@ -100,225 +122,247 @@ jQuery(document).ready(function() {
+	$('#diagnosis_step').on('click', function () {
 		prescription_json = new Object();
 		var data = $('#diagnosis').serializeArray();
 
-		if (!verifyAllFilled(data)){
+		if (!verifyAllFilled(data)) {
 			alert('Please fill out all the fields correctly before proceeding.');
-		}
-
-		else{
+		} else {
 			prescription_json.issued_on = new Date().toLocaleDateString();
-			prescription_json.id = "12345";
-    		prescription_json.symptoms = data[0].value;
-    		prescription_json.diagnosis = data[1].value;
+			prescription_json.id = '12345';
+			prescription_json.symptoms = data[0].value;
+			prescription_json.diagnosis = data[1].value;
 
-	    	loadNextStep('#diagnosis');
-	    }
+			loadNextStep('#diagnosis');
+		}
 	});
 
 	// medications step
-	$('#medications_step').on('click', function() {
+	$('#medications_step').on('click', function () {
 		var data = $('#medications').serializeArray();
 
-		if (!(verifyAllFilled(data) && data.length  == 4 * $('#medicine_group .medicine').length)){
+		if (
+			!(
+				verifyAllFilled(data) &&
+				data.length == 4 * $('#medicine_group .medicine').length
+			)
+		) {
 			alert('Please fill out all the fields correctly before proceeding.');
-		}
-
-		else{
+		} else {
 			prescription_json.medicines = [];
 
-	    	for (i = 0; i < data.length; i = i + 4){
-	    		var medicine_json = new Object();
-	    		medicine_json.name = data[i].value;
-	    		medicine_json.duration = data[i+1].value;
-	    		medicine_json.frequency = data[i+2].value;
-	    		medicine_json.usage = data[i+3].value;
-	    		prescription_json.medicines.push(medicine_json);
-	    	}
-
-	    	loadNextStep('#medications');
-	    }
-	});
+			for (i = 0; i < data.length; i = i + 4) {
+				var medicine_json = new Object();
+				medicine_json.name = data[i].value;
+				medicine_json.duration = data[i + 1].value;
+				medicine_json.frequency = data[i + 2].value;
+				medicine_json.usage = data[i + 3].value;
+				prescription_json.medicines.push(medicine_json);
+			}
 
+			loadNextStep('#medications');
+		}
+	});
 
 	// previous step
-	$('.msf-form form .btn-previous').on('click', function() {
-		$(this).parents('fieldset').fadeOut(400, function() {
-			$(this).prev().fadeIn();
-			scroll_to_class('.msf-form');
-		});
+	$('.msf-form form .btn-previous').on('click', function () {
+		$(this)
+			.parents('fieldset')
+			.fadeOut(400, function () {
+				$(this).prev().fadeIn();
+				scroll_to_class('.msf-form');
+			});
 	});
 
 	// Add step
 	$('.msf-form form .btn-plus').on('click', function () {
 		medicine_count++;
-        var html = '';
-        html += '<div class="medicine"> \
+		var html = '';
+		html +=
+			'<div class="medicine"> \
 					<div class="form-group"> \
 			            <label>Medicine Name:</label><br> \
-			            <input type="text" name="medicine_name_'+ medicine_count + '" class="form-control" required> \
+			            <input type="text" placeholder="ex-Paracetamol" name="medicine_name_ ' +
```

Comment:
Please change the placeholder to:
"Syp Ambrodil (2 tsp)"
So that the doctors will know that they need to add extra dosage information like this.